app/lib/actions/katello/system/create.rb

```diff
@@ -63,6 +65,16 @@ def finalize
           system.uuid = input[:uuid]
           system.save!
         end
+
+        def connect_to_smart_proxy(system)
+          smart_proxy = SmartProxy.where(:name => system.name).first
+
+          if smart_proxy
+            smart_proxy.system_id = system.id
```

Comment:
Shouldn't this be content_host_id?  (You can also just do  `smart_proxy.content_host = system`)